OpenKeychain/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:25.3.1'
     compile 'com.android.support:cardview-v7:25.3.1'
     compile 'com.android.support:support-annotations:25.3.1'
+    compile 'com.android.support:multidex:1.0.1'
```

Comment:
this is pointless if we don't inherit our application class from `MultiDexApplication`. it's also only needed to build with multidex for minsdk 20 or lower, so we can just drop this.

Reply:
I apparently did not read the documentation thoroughly enough, thanks.